libtest.sh

```diff
@@ -0,0 +1,356 @@
+#!/usr/bin/env bash
```

Comment:
This file that you copied over isn't from the latest onet. The latest version has comments on all the functions.